app/src/main/java/woowacourse/movie/presentation/view/main/booklist/BookListFragment.kt

```diff
@@ -0,0 +1,28 @@
+package woowacourse.movie.presentation.view.main.booklist
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.domain.Reservation
+import com.example.domain.ReservationRepository
+import woowacourse.movie.databinding.FragmentBookListBinding
+
+class BookListFragment : Fragment() {
+    private lateinit var binding: FragmentBookListBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentBookListBinding.inflate(inflater, container, false)
```

Comment:
```suggestion
        binding = FragmentBookListBinding.inflate(inflater, container, false)
```
아직 뷰바인딩 혹은 데이터바인딩에 대한 학습이 진행되지 않은 것으로 알고 있습니다.
바인딩을 제거해주시고 findViewById로 변경 부탁드립니다. 🙏  (이하 관련 내용 동일)